src/loaders/ObjectLoader.js

```diff
@@ -458,6 +458,12 @@ Object.assign( ObjectLoader.prototype, {
 
 			var clip = AnimationClip.parse( json[ i ] );
 
+			if ( json[ i ].uuid !== undefined ) {
+
+				clip.uuid = json[ i ].uuid;
+
+			}
+
```

Comment:
Please write the method like this now:
```js
parseAnimations: function ( json ) {

	var animations = [];

	for ( var i = 0; i < json.length; i ++ ) {

		var data = json[ i ];

		var clip = AnimationClip.parse( data );

		if ( data.uuid !== undefined ) clip.uuid = data.uuid;

		animations.push( clip );

	}

	return animations;

},
```
In this way it's more similar to others in `ObjectLoader`.